app/src/main/sqldelight/ca/llamabagel/transpo/data/db/RecentPlace.sq

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE recent_place (
+    id TEXT UNIQUE PRIMARY KEY,
+    primary_text TEXT NOT NULL,
+    secondary_text TEXT NOT NULL,
+    latitude TEXT NOT NULL,
+    longitude TEXT NOT NULL,
+    date INTEGER NOT NULL
+);
+
+insert:
+REPLACE INTO recent_place(id, primary_text, secondary_text, latitude, longitude, date)
+VALUES(:id, :primaryText, :secondaryText, :latitude, :longitude, :date);
+
+selectRecentPlaces:
```

Comment:
use get terminology instead of select. The function will be consumed in Kotlin, not SQL.